src/main/java/com/celements/velocity/CelementsVelocityService.java

```diff
@@ -0,0 +1,70 @@
+package com.celements.velocity;
+
+import static com.google.common.base.Preconditions.*;
+
+import java.io.StringWriter;
+
+import org.apache.velocity.VelocityContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.xwiki.component.annotation.Component;
+import org.xwiki.component.annotation.Requirement;
+import org.xwiki.velocity.VelocityManager;
+import org.xwiki.velocity.XWikiVelocityException;
+
+import com.celements.model.context.ModelContext;
+import com.celements.model.util.ModelUtils;
+import com.google.common.base.Strings;
+import com.xpn.xwiki.doc.XWikiDocument;
+
+@Component
+public class CelementsVelocityService implements VelocityService {
```

Comment:
"Celements" is already in the package name. Call it VelocityService and VelocityServiceRole (for interface) or similar.